react/FilterNavigator.js

```diff
@@ -86,6 +88,7 @@ const FilterNavigator = ({
   navigationTypeOnMobile = 'page',
   updateOnFilterSelectionOnMobile = false,
   showClearByFilter = false,
+  showClearAllFilters = false,
```

Comment:
Since it is only on the desktop, I think we should make it clear.
Maybe we should change the prop name to `showClearAllFiltersOnDesktop`